core/controllers/admin.py

```diff
@@ -412,10 +412,16 @@ def _load_dummy_new_structures_data(self):
             # for published stories.
             self._reload_exploration('15')
             self._reload_exploration('25')
+            self._reload_exploration('13')
+
             story = story_domain.Story.create_default_story(
-                story_id, 'Dummy Story 1', topic_id_1)
+                story_id, 'Help Jaime win the Arcade', topic_id_1)
             story.add_node(
```

Comment:
SHould we use loop here? (to avoid lot of duplicate steps.)

Reply:
Since this is pseudo test code, I think I would prefer to keep this section DAMP over DRY. Also, each step is not identical so the loop would be needlessly complex.

Reply:
This isn't test code, right? and `_load_dummy_new_structures_data` already looks so big so I would prefer to do it with aloo, any thoughts? 

Re complexity:

You can create a dict or llist(tuple(chapter_name, description) and oo over it? will that be helpful?

[Note: You can do these changes in a follow-u R and get this in for release.]